LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
+Rights to distribute the Mod must be granted in a separate license which must be explicitly obtained from the Author.
```

Comment:
You're not hearing what I'm saying. I'm saying that requiring someone to obtain permission before distributing the Mod is sufficient. If the author decides to do that via a distribution license...so be it. If the author doesn't want to bother with a secondary license, this will allow them to do it by simply saying to someone "go ahead, you can distribute my mod". The default state is the same...and it still requires permission, which satisfies the #StopModReposts goal, but it also avoids tacking on needless extra clauses or legalese that may confuse people who already struggle to understand licensing.